### PR TITLE
Added metrics for volume and modified  disk activity metrics

### PR DESCRIPTION
**Pull request by MohammedSentry**
Added metrics for measuring volume size and usage. Modified metrics to measure disk activity.

### CODE REVIEW

File: src/main/connector/system/Linux/Linux.yaml [hunk 1]
```diff
@@ -207,27 +196,40 @@ monitors:
...
     simple:
-      type: multiInstance
       sources:
-        fileSystemInfo:
-          # filesystem;mountpoint;type;used;available
-          type: commandLine
-          commandLine: /usr/bin/df -B1 --output=source,target,fstype,used,avail,size
+        # Name;DiskReadBytesPersec;DiskWriteBytesPersec;DiskReadsPersec;DiskWritesPersec;PercentDiskTime;PercentDiskReadTime;PercentDiskWriteTime
+        fileSystemInformation:
+          type: wmi
```
Comment (bertysentry) [inline src/main/connector/system/Linux/Linux.yaml:202]: This probably has nothing to do in the  Linux connector!

File: src/main/connector/system/Windows/Windows.yaml [hunk 2]
```diff
@@ -267,8 +256,8 @@ monitors:
         attributes:
           id: $1
         metrics:
-          system.disk.io{disk.io.direction="read"}: $2
-          system.disk.io{disk.io.direction="write"}: $3
+          system.disk.operations.bytes{direction="read"}: $2
```
Comment (bertysentry) [inline src/main/connector/system/Windows/Windows.yaml:259]: I'm sorry but this is breaking compatibility with [OpenTelemetry semantic conventions](https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systemdiskio)

Please keep `system.disk.io`. Similarly, other changes in metric names are breaking compatibility and must be fixed before we can merge.

**Review by bertysentry — CHANGES_REQUESTED** [review]
Some changes are breaking compatiblity with OpenTelemetry semantic conventions.

File: src/main/connector/system/Windows/Windows.yaml (hunk 2 above)
Comment (MohammedSentry) [reply src/main/connector/system/Windows/Windows.yaml:259]: @bertysentry thanks for the review.
If we keep the metric `system.disk.io`, is it possible to have bytes as the unit?
Or must the metric necessarily be in IO?

File: src/main/connector/system/Linux/Linux.yaml [hunk 3]
```diff
@@ -143,34 +143,23 @@ monitors:
...
+                print $3 ";" $6*512 ";" $10*512
+              }
       mapping:
-        source: ${source::physicalDiskInformation}
+        source: ${source::physicalDiskActivity}
         attributes:
           id: $1
-          system.disk.device: $1
         metrics:
-          system.disk.operations{direction:read}: $2
```
Comment (bertysentry) [inline src/main/connector/system/Linux/Linux.yaml:167]: Why are we removing these metrics? They are not available? (`system.disk.operations`, etc.)

File: src/main/connector/system/Linux/Linux.yaml [hunk 4]
```diff
@@ -210,24 +199,20 @@ monitors:
...
           id: $1
-          system.filesystem.device: $1
-          system.filesystem.mountpoint: $2
-          system.filesystem.type: $3
         metrics:
-          system.filesystem.usage{system.filesystem.state=used}: $4
-          system.filesystem.usage{system.filesystem.state=free}: $5
-          system.filesystem.utilization{system.filesystem.state=used}: $6
-          system.filesystem.utilization{system.filesystem.state=free}: $7
+          system.filesystem.size.bytes: $2
```
Comment (bertysentry) [inline src/main/connector/system/Linux/Linux.yaml:212]: This doesn't follow OpenTelemetry semantic conventions. Please make sure to follow [these conventions](https://opentelemetry.io/docs/specs/semconv/system/system-metrics/).

File: src/main/connector/system/Windows/Windows.yaml [hunk 5]
```diff
@@ -267,8 +256,8 @@ monitors:
         attributes:
           id: $1
         metrics:
-          system.disk.io{disk.io.direction="read"}: $2
-          system.disk.io{disk.io.direction="write"}: $3
+          system.disk.io{direction="read"}: $2
```
Comment (bertysentry) [inline src/main/connector/system/Windows/Windows.yaml:259]: `system.disk.io` is correct. Please revert. (see https://opentelemetry.io/docs/specs/semconv/system/system-metrics/#metric-systemdiskio)

File: src/main/connector/system/Windows/Windows.yaml [hunk 6]
```diff
@@ -124,50 +124,39 @@ monitors:
...
-      mapping:
-        source: ${source::serviceInformation}
-        attributes:
-          id: $1
-          description: $3
+          name: $2
+          label: $3
         metrics:
-          system.service.status: $2
+          system.filesystem.size.bytes: $4
```
Comment (bertysentry) [inline src/main/connector/system/Windows/Windows.yaml:156]: Not OpenTelemetry compliant

File: src/main/connector/system/System/System.yaml [hunk 7]
```diff
@@ -126,4 +126,16 @@ metrics:
   system.uptime:
     description: System uptime in seconds indicates how long the host has been running since its last boot
     type: Gauge
-    unit: s
\ No newline at end of file
+    unit: s
+  system.volume.size.bytes:
```
Comment (bertysentry) [inline src/main/connector/system/System/System.yaml:130]: These are no longer required, as everything is now exposed with `system.filesystem` metrics, right?

**Review by bertysentry — APPROVED** [review]

**Pull request merged**